templates/wsgi.py.epp

```diff
@@ -1,13 +1,12 @@
+# THIS FILE IS MANAGED BY PUPPET
```

Comment:
There are no variables left. Please move it to `files/wsgi.py` and use `file('puppetboard/wsgi.py')`